plugins/polio/serializers.py

```diff
@@ -356,6 +356,9 @@ def create(self, validated_data):
         destructions = validated_data.pop("destructions", [])
         started_at = validated_data.get("started_at", None)
         ended_at = validated_data.get("ended_at", None)
+        datelogs = validated_data.get("datelogs", None)
+        if datelogs:
+            raise serializers.ValidationError({"datelogs": "Cannot have modification history for new round"})
```

Comment:
How could this have happened previously? Why is this needed?

Reply:
It could happen previously because the UI would let users add a `datelog` when they would create a round 0 in a campaign that has already other rounds. 

the PR fixes the problem front-end side. I just added the check in the backend for good measure